src/Extension/SiteConfigExtension.php

```diff
@@ -43,30 +41,27 @@ public function updateCMSFields(FieldList $fields)
         Requirements::javascript('silverstripe/mfa: client/dist/js/bundle-cms.js');
         Requirements::css('silverstripe/mfa: client/dist/styles/bundle-cms.css');
 
-        $mfaEnabled = CheckboxField::create(
-            'MFAEnabled',
-            _t(__CLASS__ . '.MFA_ENABLED', 'Enable MFA for CMS access')
-        );
-        $mfaEnabled->addExtraClass('mfa-settings__enabled');
-
         $mfaOptions = OptionsetField::create(
             'MFARequired',
             '',
             [
                 false => _t(__CLASS__ . '.MFA_OPTIONAL', 'MFA is optional for everyone'),
                 true => _t(__CLASS__ . '.MFA_REQUIRED', 'MFA is required for everyone'),
-
             ]
         );
-        $mfaOptions->addExtraClass('mfa-settings__required mfa-settings--hidden');
+        $mfaOptions->addExtraClass('mfa-settings__required');
 
         $mfaGraceEnd = DateField::create(
             'MFAGracePeriodExpires',
             'Optional: grace period end date when MFA is enforced'
         );
+        $mfaGraceEnd->setDescription(_t(
+            __CLASS__ . '.MFA_GRACE_DESCRIPTION',
+            'MFA setup will be optional during this grace period, and reminders to set up will appear at log in.'
```

Comment:
Possible improvement: "and reminders to register will appear at log in."

Reply:
I was thinking that as I typed it, but was also aware that changes were inbound from PUX, so didn't think about it too much. The more one thinks about it the more and more "wordy" it has the propensity to become :upside_down_face: